test/index.browser-test.js

```diff
@@ -0,0 +1,138 @@
+import jsdom from 'jsdom';
+import { assert } from 'chai';
+import sinon from 'sinon';
+import ReactDOM from 'react-dom';
+import ifReact from 'enzyme-adapter-react-helper/build/ifReact';
+
+import AphroditeComponent from './components/AphroditeComponent';
+import {
+  renderReactWithAphrodite as serverRenderReactWithAphrodite,
+  renderReactWithAphroditeStatic as serverRenderReactWithAphroditeStatic,
+} from '..';
+import {
+  renderReactWithAphrodite,
+  renderReactWithAphroditeStatic,
+} from '../lib/index.browser';
+
+describe('renderReactWithAphrodite aphrodite css rendering', () => {
+  let originalWindow;
+  let originalDocument;
+  let result;
+
+  beforeEach(() => {
+    originalWindow = global.window;
+    originalDocument = global.document;
+    result = serverRenderReactWithAphrodite('AC', AphroditeComponent)({
+      children: ['Zack'],
+      onPress() { console.log('Clicked'); },
+    });
+  });
+
+  afterEach(() => {
+    global.window = originalWindow;
+    global.document = originalDocument;
+  });
+
+  it('the markup looks good', () => {
+    assert.isString(result);
+
+    assert.ok(/style data-aphrodite/.test(result));
+    assert.ok(/Zack/.test(result));
+    assert.ok(/data-aphrodite-css/.test(result));
+  });
+
+  it('throws when calling renderReactWithAphrodite on the server', () => {
+    assert.throws(() => renderReactWithAphrodite('AC', AphroditeComponent), 'functionality does not work');
```

Comment:
@goatslacker For some reason, this doesn't actually throw, and I'm not sure why. Can you help me figure this out?